nservicebus/hosting/expose-an-endpoint-as-a-web-wcf-service.md

```diff
@@ -10,6 +10,8 @@ related:
  - samples/web/wcf-callbacks
 ---
 
+Note: NServiceBus Version 6 and above no longer provide built-in support for WCF integration. Refer to the [Upgrade Guide](/nservicebus/upgrades/host-6to7) for further information.
```

Comment:
I think this should be warning instead. And a minor suggestion:

WARNING: Starting from NServiceBus Core Versions 6 and above, the built-in support for WCF integration has been removed.  For more details on how to use WCF integration for these versions, refer to the [Upgrade Guide](/nservicebus/upgrades/host-6to7).